baggers.js

```diff
@@ -14357,12 +14417,24 @@ var data = {
       lat: 45.764043,
       lng: 4.835659,
     },
+    Luxembourg: {
+      name: 'Luxembourg',
+      ville_img: 'img/villes/BBL_Luxembourg.png',
```

Comment:
You need to add this file in https://github.com/brownbaglunch/BrownBagLunch/tree/gh-pages/img/villes